source/_integrations/unifi.markdown

```diff
@@ -152,6 +152,13 @@ Get entities reporting the general temperature of a UniFi Network device.
 
 Get entities reporting the current state of a UniFi Network device.
 
+### WLAN Password sensor
+
+Provides the current password for a specific WLAN. This sensor allows users to retrieve the WLAN password for various purposes, such as sharing with guests or configuring devices manually.
```

Comment:
I would like to clarify that this is more suggested as a fallback when the QR code is not applicable

Reply:
Check my adjusted version.